Fix typo parsing module reqs

r4f4 opened the pull request:
Signed-off-by: Rafael Fonseca <r4f4rfs@gmail.com>

igor-raits commented:
@r4f4 does libmodulemd reject stream list like `[-1, 2]`? Or is it just that people SHOULD NOT do that because it is pointless?

r4f4 commented:
@ignatenkobrain right now it doesn't reject but it's a bug in libmodulemd according to @sgallagher.

igor-raits commented:
once it does, let's bump version of libmodulemd and do this.

igor-raits commented on main.c:
```diff
@@ -45,22 +45,17 @@ parse_module_requires (Pool       *pool,
-
-          Id *r;
           if (s[0] == '-')
             {
```
remove these

r4f4 commented:
Ok, so I'll update the patch to just fix the typo for now.

sgallagher commented:
> @ignatenkobrain right now it doesn't reject but it's a bug in libmodulemd according to @sgallagher.

Yes, libmodulemd should be validating this input. I'm going to be reworking that interface in API 2.0 to make it so that it's rejected properly. @r4f4 please open an issue in https://github.com/fedora-modularity/libmodulemd/issues

r4f4 committed:
Fix typo when parsing module requires

We want to remove the '-' from the require name, not skip the next
require in the list.

Signed-off-by: Rafael Fonseca <r4f4rfs@gmail.com>

r4f4 commented:
ping
